rn/Teacher/ios/TeacherE2ETests/DSPeopleE2ETests.swift

```diff
@@ -0,0 +1,62 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import TestsFoundation
+
+struct PeopleListCell {
+    let index: Int
+    let idPrefix = "people-list-cell-row-"
+    lazy var cell: Element = {
+        return app.find(id: "\(idPrefix)\(index)")
+    }()
+
+    lazy var name: String = {
+        return app.find(id: "\(idPrefix)\(index).name-label").label()
```

Comment:
Is `lazy` a good idea here? If we call this method multiple times it will always return the value of the first invocation.

Reply:
Agreed, and done.